config/tests/samples/create/harness.go

```diff
@@ -529,11 +529,20 @@ func (h *Harness) GetClient() client.Client {
 	return h.client
 }
 
+func (h *Harness) GetRESTConfig() *rest.Config {
+	return h.restConfig
+}
+
 func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured) {
 	if os.Getenv("E2E_GCP_TARGET") == "mock" {
 		for _, resource := range resources {
 			gvk := resource.GroupVersionKind()
 
+			// Special fake types for testing
+			if gvk.Group == "" && gvk.Kind == "RunCLI" {
```

Comment:
Not a blocker but on the remote chance that this ever leaked onto a real cluster I would prefer a more obvious kind (eg. RunTestCLI).